docs/ADRs/0005-python-tooling.md

```diff
@@ -0,0 +1,43 @@
+---
+    title: 0005 Python coding standard and tools
+    adr:
+        author: Berry den Hartog
+        created: 28-Feb-2024
+        status:  accepted
+---
+
+## Context
+
+In modern software development, maintaining code quality is crucial for readability, maintainability, and collaboration.
+Python, being a dynamically typed language, requires robust tooling to ensure code consistency and type safety. Manual
+enforcement of coding standards is time-consuming and error-prone. Hence, adopting automated tooling to streamline this
+process is imperative.
+
+## Decision
+
+We will use these standards and tools for our own projects:
+
+- [Google style guide](https://google.github.io/styleguide/pyguide.html)
+- [Ruff](https://docs.astral.sh/ruff)
+    - [Rules](https://docs.astral.sh/ruff/rules/): [I, SIM, B, UP, F, E]
```

Comment:
❤️ (didn't know `SIM`, nice!)

Reply:
I read the site, these are the most commonly used (without being to intens)